specification/compute/resource-manager/readme.md

```diff
@@ -47,36 +46,6 @@ input-file:
 - Microsoft.Compute/2017-03-30/compute.json
 - Microsoft.Compute/2017-03-30/disk.json
 - Microsoft.Compute/2017-03-30/runCommands.json
-- Microsoft.ContainerService/2017-01-31/containerService.json
-```
-
-### Tag: package-compute-2017-03
-
-These settings apply only when `--tag=package-compute-2017-03` is specified on the command line.
-
-``` yaml $(tag) == 'package-compute-2017-03'
-input-file:
-- Microsoft.Compute/2017-03-30/compute.json

```

Comment:
Just want to know the intention here, are you sure you want to delete compute-specific tags even though most changes here are related to `ContainerServices`

Reply:
This particular tag is identical to the `package-2017-03` tag above it once `containerService.json` was moved, so I removed it.